### PR TITLE
[14.0][MIG] analytic_base_department

### PR DESCRIPTION
**Pull request by Saran440**
Standard Migrate and cherry-pick from https://github.com/OCA/account-analytic/pull/338

@rafaelbn I think this module still used because
- Module hr_timesheet is not necessary install for all project
- There is field `department_id` on `account_analytic_line` when install module `hr_timesheet` BUT it not display.
- In core odoo, `account_analytic` without department_id

### CODE REVIEW

File: analytic_base_department/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2011-2018 Camptocamp SA
+# Copyright 2013 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Analytic Department Categorization",
```
Comment (MeetKD) [inline analytic_base_department/__manifest__.py:5]: @Saran440 It would be great if you can add summary about the module !

Comment (Saran440) [reply analytic_base_department/__manifest__.py:5]: Okay I will. Thank you for your reviewed. :)

**Review by gurneyalex — APPROVED** [review]

File: analytic_base_department/__manifest__.py (hunk 1 above)
Comment (Saran440) [reply analytic_base_department/__manifest__.py:5]: @MeetKD I added it. Please, Could you review and approved this?

**Pull request merged**